### DISCUSSION TITLE
Is there are way to provision Kedro Viz with subpath

### DISCUSSION THREAD

**Discussion by parachill** [kedro-org/kedro#2063] [Idea]
I want url like http://127.0.0.1:4141/xxxx is there are way to do this?

Thanks,

**Comment by stichbury** [comment]
Hi @parachill! Please can you raise your question over on our Slack organisation (https://slack.kedro.org/) if you've not already done so. That's the place for questions about Kedro now and you'll get a response much more swiftly.
